openmdao/docs/feature_reference/core_features/working_with_derivatives/approximating_totals.rst

```diff
@@ -0,0 +1,52 @@
+.. _feature_declare_totals_approx:
+
+
+Approximating Semi-Total Derivatives
+====================================
+
+There are also times where it makes more sense to approximate the derivatives for an entire group in one shot. You can turn on
+the approximation by calling `approx_totals` on any `Group`.
+
+.. automethod:: openmdao.core.group.Group.approx_totals
+    :noindex:
+
+The default method is for approximating semi-total derivatives is the finite difference method. When you call the `approx_totals` method on a group, OpenMDAO will
```

Comment:
extraneous "is":  "The default method is for..."

Reply:
Fixed